revscoring/languages/persian.py

```diff
@@ -10,11 +10,43 @@
     raise ImportError("No enchant-compatible dictionary found for 'fa'.  " +
                       "Consider installing 'myspell-fa'.")
 
+REVERENTWORDS = set([
```

Comment:
we don't have features related to this regex, it needs lots and lots of upstream coding which seems unnecessary to me. Just merge it with informal words.